Sources/StreamChat/Controllers/BackgroundListDatabaseObserver.swift

```diff
@@ -158,12 +166,14 @@ class BackgroundListDatabaseObserver<Item, DTO: NSManagedObject> {
             cacheName: nil
         )
 
+        _ = processingQueue
```

Comment:
what's this?

Reply:
Can't we simply pass the `processingQueue` in the constructor? This solution is a bit confusing. Besides that, I would try to avoid lazy initializations since we had concurrency issues already with this approach.

Reply:
`processingQueue` is internal details, so it should not be passed from the outside ideally. This approach makes the definition of the queue be in one place, whilst making it thread-safe.

I can move its definition to the init, but that will pollute it a bit.

Reply:
The problem here is that at first sight `_ = processingQueue` is not clear why this is needed. I would rather configure the queue in the init then if we don't want to pass it from the outside.